selfservice/hook/verification.go

```diff
@@ -80,6 +90,16 @@ func (e *Verifier) do(r *http.Request, i *identity.Identity, f flow.Flow) error
 			return err
 		}
 
+		flowURL := ""
+		if verificationFlow.Type == flow.TypeBrowser {
+			flowURL = verificationFlow.AppendTo(e.r.Config().SelfServiceFlowVerificationUI(r.Context())).String()
+		}
+
+		f.AddContinueWith(flow.NewContinueWithVerificationUI(verificationFlow, address.Value, flowURL))
+
+		if err := e.r.PrivilegedIdentityPool().UpdateVerifiableAddress(r.Context(), address); err != nil {
```

Comment:
Why do we need this? There have been no changes to the address as far as I can tell?

Reply:
True, removed.